test/jit/test_monkeytype.py

```diff
@@ -0,0 +1,13 @@
+import torch
+
+def test_sum_monkey(a, b):
```

Comment:
These functions can be nested functions inside test case, then use `make_global` to expose them to global name space. That would provide better readability and isolation.

Reply:
I have moved everything to `test_pdt.py`. So I guess this is not required

Reply:
Fixed now.